Makefile

```diff
@@ -409,6 +409,11 @@ providers/build/tailscale: providers/lr
 providers/install/tailscale:
 	@$(call installProvider, providers/tailscale)
 
+providers/build/ipinfo: providers/lr
```

Comment:
we also need to include this in the make tasks `providers/build`, `providers/install`, `providers/dist`, `providers/bundle`, `providers/test` and `lr/docs/markdown`